### PR TITLE
correct 32bit timestamp from euCliTeleReader, and aidastrip dev update.

### PR DESCRIPTION
**Pull request by MengqingWu**
- rebased to the current master then commit. 
- except for : user/eudet/exe/src/euCliTeleReader.cxx 
- Other changes only in user/aidastrip/ folder

### CODE REVIEW

**Comment by lhuth** [comment]
The change in the euCliReader.cxx should also adjust the boost option, which is still 32bit

**Comment by MengqingWu** [comment]
> The change in the euCliReader.cxx should also adjust the eucaq::option, which is still 32bit. See line 13/14

Done.

**Comment by MengqingWu** [comment]
> The change in the euCliReader.cxx should also adjust the eucaq::option, which is still 32bit. See line 13/14

Done.

**Comment by lhuth** [comment]
I am happy with the PR as is - Now @eyiliu or @dreyling also need to check it as I cannot merge stuff myself

File: main/exe/src/euCliReader.cxx [hunk 1]
```diff
@@ -11,8 +11,8 @@ int main(int /*argc*/, const char **argv) {
   eudaq::Option<uint32_t> eventh(op, "E", "eventhigh", 0, "uint32_t", "event number high");
   eudaq::Option<uint32_t> triggerl(op, "tg", "trigger", 0, "uint32_t", "trigger number low");
   eudaq::Option<uint32_t> triggerh(op, "TG", "triggerhigh", 0, "uint32_t", "trigger number high");
-  eudaq::Option<uint32_t> timestampl(op, "ts", "timestamp", 0, "uint32_t", "timestamp low");
-  eudaq::Option<uint32_t> timestamph(op, "TS", "timestamphigh", 0, "uint32_t", "timestamp high");
+  eudaq::Option<uint64_t> timestampl(op, "ts", "timestamp", 0, "uint32_t", "timestamp low");
+  eudaq::Option<uint64_t> timestamph(op, "TS", "timestamphigh", 0, "uint32_t", "timestamp high");
```
Comment (eyiliu) [inline main/exe/src/euCliReader.cxx]: Both uint32_t should be moved to uint64.
` eudaq::Option<uint64_t> timestamph(op, "TS", "timestamphigh", 0, "uint32_t", "timestamp high");
`

` eudaq::Option<uint64_t> timestamph(op, "TS", "timestamphigh", 0, "uint64_t", "timestamp high");
`
Also in the file euCliTeleReader.cxx

Comment (MengqingWu) [reply main/exe/src/euCliReader.cxx]: Done.

**Comment by eyiliu** [comment]
Is your change ready to merge?

**Comment by eyiliu** [comment]
@MengqingWu  
Closed. Suggest to rebase the commits history and open a new pull request.

**Pull request closed without merging**